django/contrib/admin/templates/admin/change_form.html

```diff
@@ -73,15 +73,39 @@
             $(document).ready(function() {
```

Comment:
I'm curious if we should have this JS code in here at all. Wouldn't it make more sense to move it into a separate .js file?